### PR TITLE
TINY-4728: Fix table picker breaking in Firefox on low zoom levels

### LINKED ISSUE #5430: Table picker is broken when zoom out
**What is the current behavior? Describe the bug**
Open table picker plugin and try to create a table, you'll see the picker is not functioning.
image

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**
Only happens on **Firefox**, (I tested on version 51 and 72), zoom out to 90% or lower. then go to https://www.tiny.cloud/docs/demo/full-featured/ click table and then starting pick table
image

**What is the expected behavior?**
Should not broken.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
Tested on 5.1.0 and current cloud version on your website.

### PR DESCRIPTION
**Pull request by SimonFc**
Quick fix, this makes the table picker use grid layout whenever possible.
Fixes #5430

### CODE REVIEW

**Comment by lnewson** [comment]
I'm not sure I like the idea of IE, etc... using something different as it's likely to cause a lot of maintenance overhead/issues when making updates. Did you try the different box sizing mentioned in https://stackoverflow.com/a/1274647 (from the BZ link), as that seems like a nicer solution if it works?

**Comment by lnewson** [comment]
Okay, I managed to reproduce it on Mac but not until you get to 40% zoom. I tested making the buttons use `box-sizing: border-box` and that does appear to fix it, though I had to make the following additional changes:
```
@insert-table-picker-cell-size: @base-value + 1; // Add 1 for the border size

  .tox-insert-table-picker {
    width: @insert-table-picker-cell-size * 10 + 10px; // Add one to account for the different box model, plus there's 10 boxes, so 10 borders.
  }
```

If we don't want to do that and instead want to stick with the grid, then I really think we need to remove the `@supports (display: grid) {` and have it working for all supported browsers.

**Comment by jhaines** [comment]
I feel we shouldn't be using `@supports` when we've used `display: grid` elsewhere without it.  Not overly concerned about using something else over `display: grid` though.

**Comment by lnewson** [comment]
I'm also not worried about whether we use CSS grids or the border-box approach. However atm it's a halfway point where it's trying to do both, so we should pick and stick with it. :woman_shrugging:

**Comment by spocke** [comment]
Right we where discussing this yesterday and we thought IE doesn't have grid support but know I remember that it does have that but a bit more limited but maybe enough for us to do what we need here.

File: modules/tinymce/changelog.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Version 5.3.0 (TBD)
     Changed the `link`, `image` and `paste` plugins to use Promises to reduce the bundle size #TINY-4710
     Fixed the `quickimage` button not restricting the file types to images #TINY-4715
+    Fixed table picker breaking in Firefox on low zoom levels
```
Comment (lnewson) [inline modules/tinymce/changelog.txt]: Do we have a JIRA number for this?

**Comment by SimonFc** [comment]
@lnewson Thanks for your alternative solution! It's much simpler and avoids the branching

**Pull request merged**